playbook2.yaml

```diff
@@ -0,0 +1,99 @@
+- name: Projex4 - run your To-Do app on the managed VM
+  hosts: managedNodes
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: TRELLO_BOARD_ID
+      prompt: Please enter Trello Board ID >
+      default: 64ae6f476f946f8c451a5cb2
+      private: false
+
+    - name: TRELLO_TODO_ID
+      prompt: Please enter Trello TODO List ID >
+      default: 64ae6f476f946f8c451a5cb9
+      private: false
+
+    - name: TRELLO_DOIN_ID
+      prompt: Please enter Trello DOIN List ID >
+      default: 64ae6f476f946f8c451a5cba
+      private: false
+
+    - name: TRELLO_DONE_ID
+      prompt: Please enter Trello DONE List ID > 
+      default: 64ae6f476f946f8c451a5cbb
+      private: false
+
+    - name: TRELLO_APIKEY
+      prompt: Please enter Trello API key > 
+      #default: 7d069c46c188afd7a7eaeb18851b9fac #sensitive so 'not' committing to git
+      private: true
+
+    - name: TRELLO_TOKEN
+      prompt: Please enter Trello token >
+      #default: ATTAcff8cd91ef9d8108a296d4954de006c36a5f27507477246e8cd87d76edbdf881F66B1476 #sensitive so 'not' committing to git
```

Comment:
...Oops!

We should remove this from the file now, but more importantly we should treat it as compromised and [revoke the token](https://developer.atlassian.com/cloud/trello/guides/rest-api/authorization/#:~:text=Users%20are%20able%20to%20revoke,be%20deleted%20via%20the%20API.) and generate a new one afterwards for us going forwards